docs/company/getting-started.md

```diff
@@ -0,0 +1,35 @@
+# Getting Started at Thoth Tech
+
+Welcome to Thoth Tech!
+
+If you are reading this Getting Started Guide, then you have just enrolled in the Thoth Tech Company and we welcome you! 
+
+This guide is a place to introduce you to the company and all the resources availible to help get you onboarded, introduce you to the comapny and products, and skilled up to tackle the Trimester ahead.
+
+##Who are we?
+
+The best place to start to familiarise yourself with Thoth Tech is the [company charter](docs/company/../../charter.md). This is an important guide for understanding the mission and values of the company which informs the way that the company runs and is crucial to ensuring your experience is the best it can be.
```

Comment:
Slight reword:

```suggestion
The best place to start to familiarise yourself with Thoth Tech is the [company charter](docs/company/../../charter.md). This is important for understanding the mission and values of the company which inform the way the company runs. Understanding and employing this charter is crucial to ensuring your, and others, experience is the best it can be.
```